migrations/versions/2022_041211_5269e8c2c2c9_.py

```diff
@@ -0,0 +1,82 @@
+"""Create partner tables and add the partner_id column to users
+
+Revision ID: 5269e8c2c2c9
+Revises: b500363567e3
+Create Date: 2022-04-12 11:56:30.184292
+
+"""
+import arrow
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '5269e8c2c2c9'
+down_revision = 'b500363567e3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    partner_table = op.create_table(
```

Comment:
Do you know if this migration file is auto generated? In general we should try to auto generate it as much as possible and manually editing it should the last resort.

Reply:
No no, we should never change the database by hand :D. Everytime we need to make a change, we modify models.py and let alembic generate the necessary migration files. The database generation is only called once we are sure about the model changes so the dev flow is usually like:

1. make necessary change in models.py to support the new feature
2. run new-migration.sh to generate the migration file and run the migration via `alembic upgrade head`
3. continue working on other parts of the app. If we realized that the model change isn't right here, we need to revert the change in the database by `alembic downgrade -1` and starts from step 1 over again.